programs/axelar-solana-gateway/src/error.rs

```diff
@@ -169,18 +169,18 @@ mod tests {
 
     #[test]
     fn test_should_relayer_proceed() {
-        let errors_to_proceed = (0..IRRECOVERABLE_ERROR)
+        let errors_to_proceed = (0..6)
             .map_while(GatewayError::from_u32)
             .collect_vec();
-        let errors_to_not_proceed = (IRRECOVERABLE_ERROR..u32::MAX)
+        let errors_to_not_proceed = (6..u32::MAX)
```

Comment:
Using the constant makes the test always pass whatever its value.
Once we get in prod we don't want this value to change as it would be a breaking change, hence the removal of the reference to that constant here